src/test/tpc/tpch_test.cpp

```diff
@@ -44,11 +44,16 @@ class TPCHTest : public BaseTestWithParam<TPCHTestParam> {
       {1, 0.01f},   {2, 0.004f},  {3, 0.01f},  {4, 0.005f},  {5, 0.01f},    {6, 0.01f},  {7, 0.01f},  {8, 0.01f},
       {9, 0.01f},   {10, 0.02f},  {11, 0.01f}, {12, 0.01f},  {13, 0.01f},   {14, 0.01f}, {15, 0.01f}, {16, 0.01f},
       {17, 0.013f}, {18, 0.005f}, {19, 0.01f}, {20, 0.008f}, {21, 0.0075f}, {22, 0.01f}};
+
+  // Helper method used to touch TPCHBenchmarkItemRunner's privates
+  std::string get_deterministic_query(TPCHBenchmarkItemRunner& runner, BenchmarkItemID item_id) {
+    return runner._build_deterministic_query(item_id);
```

Comment:
What's the danger of making this public?

Reply:
No danger in particular, but something that is only supposed to be used by tests seems like a candidate for a non-public interface. No strong feeling about this.